tests/python/unittest/test_operator.py

```diff
@@ -6955,7 +6955,7 @@ def test_stack():
 
 
 @with_seed()
-@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/14288")
+#@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/14288")
```

Comment:
remove?

Reply:
This test is still flaky. I commented it out only to test my PR. I suppose we should enable it after the flakiness issue is resolved?

Reply:
Can we enable the non-cudnn tests, and mark the cudnn dropout test as flaky? Otherwise this PR is not checked with unit test